specification/devcenter/DevCenter/shared/models.tsp

```diff
@@ -6,53 +6,38 @@ import "../main.tsp";
 using Azure.Core;
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
 
 namespace DevCenterService;
 
-@doc("Indicates whether operation status is running, completed, canceled or failed.")
-@lroStatus
-union OperationStatusValue {
```

Comment:
Just double checking, this wasnt generated in any SDK right? Particularly any that were released as stable already?

Reply:
Actually it looks like the service doesnt support the same terminal values as the standard OperationState type defined in Azure core. Note the following different values: completed instead of succeeded. This has to be reverted :(

Reply:
Our OperationStatusValue in the tsp was wrong. Our service actually does return succeded and not complated. OperationState matches our service lro

Reply:
you can see in the recording [here](https://github.com/Azure/azure-sdk-assets/blob/95190a8afc4f186def04729c747b3a129d96e76c/net/sdk/devcenter/Azure.Developer.DevCenter/tests/SessionRecords/DevBoxesClientTests/RestartDevBoxSucceeds.json#L546C8-L546C31)